share/functions/__fish_describe_command.fish

```diff
@@ -6,15 +6,25 @@
 # The whatis database is non-existent, so apropos tries (and fails) to create it every time,
 # which takes about half a second.
 #
-# So we disable this entirely in that case.
-if test (uname) = Darwin
-    set -l darwin_version (uname -r | string split .)
-    # macOS 15 is Darwin 19, this is an issue up to and including 10.15.3.
-    if test "$darwin_version[1]" = 19 -a "$darwin_version[2]" -le 3
-        function __fish_describe_command
+# Instead, we build a whatis database in the user home directory
+# and use grep to query it
+#
+# A function `__fish_apropos_update` is provided for updating
+#
+function __fish_apropos -d "Internal command to query the apopos database"
+    if test (uname) = Darwin
+        set db ~/.whatis.db
```

Comment:
`set -l` is better, in case someone has a universal variable with that name.. we should really change that.

Reply:
updated!